.gitignore

```diff
@@ -1 +1,19 @@
-src/bin/*.dll
```

Comment:
Please don't delete this line; even if your other patterns later cover it, it's still useful documentation.